src/App.tsx

```diff
@@ -1,29 +1,57 @@
-import { Component } from 'react';
+import React, { useState, useEffect } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import { FindMovie } from './components/FindMovie';
+import { getMovie } from './api/api';
 
-interface State {
-  movies: Movie[];
-}
+export const App: React.FC = () => {
+  const [movies, setMovies] = useState<(Movie | null)[]>([]);
+  const [input, setInput] = useState('');
+  const [movie, setMovie] = useState<Movie | null>(null);
+  const [clicked, setClicked] = useState(false);
```

Comment:
Move these to `FindMovie.tsx` component and them callback, as well.
This logic is connected with finding movies, so it shouldn't be here.